### PR TITLE
ENH: Add initial draft of random

### PR DESCRIPTION
**Pull request by bashtage**
Add random section draft

### CODE REVIEW

**Comment by jarrodmillman** [comment]
This looks good to me.  I will merge later today unless someone objects.

File: paper.tex [hunk 1]
```diff
@@ -355,6 +355,102 @@ \section*{Recent technical improvements}
...
+
+The RandomState object makes a compatibility guarantee so that a fixed seed and
+sequence of function calls produce the same set of values. This guarantee has
+slowed progress since improving the underlying code requires extending the API
+with additional keyword arguments. This guarantee continues to apply to
+RandomState. 
+
+NumPy 1.17 introduced a new API for generating random numbers that use a more
+flexible structure that can be extended by libraries or end-users. The new API
+is build using components separate the tasks needed when generating random
```
Comment (mattip) [inline paper.tex]: "is built using separate components for each of the tasks ...

File: paper.tex [hunk 2]
```diff
@@ -355,6 +355,102 @@ \section*{Recent technical improvements}
...
+initial values.
+
+Bit generators are simple classes that manage the state of an underlying
+pseudorandom number generator. NumPy ships with four bit generators. The default
+bit generator is a 64-bit implementation of the Permuted Congruential Generator
+\cite{pcg64} (\code{PCG64}). The three other bit generators are a 64-bit version
+of the Philox generator\cite{random123} (\code{Philox}), Chris Doty-Humphrey's
+Small Fast Chaotic generator\cite{practrand} (\code{SFC64}), and the 32-bit
+Mersenne Twister\cite{mt19937} (\code{MT19937}) which has been used in older
+versions of NumPy.\footnote{The randomgen project supplies a wide range of
```
Comment (mattip) [inline paper.tex]: randomgen link could be here

Edit: it is a link, just later

**Comment by mattip** [comment]
Small nits, could be fixed later

File: paper.tex (hunk 1 above)
Comment (bashtage) [reply paper.tex]: Thanks.

**Pull request merged**